packages/@uppy/companion/src/server/helpers/request.js

```diff
@@ -95,6 +121,8 @@ const getProtectedHttpAgent = ({ protocol, blockLocalIPs }) => {
   return protocol.startsWith('https') ? HttpsAgent : HttpAgent
 }
 
+module.exports.getProtectedHttpAgent = getProtectedHttpAgent
```

Comment:
as this wasnt exported before maybe now's still a good time to rename the second parameter to `allowLocalIps` or similar to streamline it more with `allowLocalUrls`

Or do you have a good idea for a common name?

Reply:
Not sure what to do here.

The property in `req.companion.options` is called `allowLocalUrls`. 
We can use that name in `validateURL` and `getProtectedHttpAgent` as they haven't been exported before - on the other hand `getProtectedGot` uses `blockLocalIPs`  has been exported before, so we can't really change it without it being a potentially breaking change?

Reply:
Thoughts?

Reply:
I'd leave it as is since `getProtectedGot` also uses it already 👍

Reply:
Alright, fine! 😎

Reply:
I agree we should rename it to make it consistent. added a todo for the next major!